classes/module/Module.php

```diff
@@ -3439,6 +3439,10 @@ protected function trans($id, array $parameters = [], $domain = null, $locale =
      */
     public function isUsingNewTranslationSystem()
     {
+        if ($this->context === null || $this->context->getTranslator() === null) {
```

Comment:
@PierreRambaud are you sure about this one? Should we integrate this modification that you needed for tests weird environment or let it explode as it's not supposed to happen?

Reply:
You can remove it and retry in local. It happened because of local settings maybe.